eth/main.cpp

```diff
@@ -406,403 +320,601 @@ int main(int argc, char** argv)
 	bool chainConfigIsSet = false;
 	string configJSON;
 	string genesisJSON;
+    po::options_description clientDefaultMode("Client mode (default)");
+    clientDefaultMode.add_options()
+		    ("format", po::value<string>(), "<format> Set format.")
+		    ("script", po::value<string>(), "<script> Add script.")
+            ("mainnet", "Use the main network protocol.")
+            ("ropsten", "Use the Ropsten testnet.")
+		    ("testnet", "Use the Ropsten testnet.")
+		    ("gas-floor", po::value<string>(), "Set gasFloor")
+            ("private", po::value<string>(), "<name> Use a private chain.")
+            ("test", "Testing mode: Disable PoW and provide test rpc interface.")
+            ("config", po::value<string>(), "<file> Configure specialised blockchain using given JSON information.")
+            ("oppose-dao-fork", "Ignore DAO hard fork (default is to participate).\n")
```

Comment:
This can be deleted

Reply:
why?

Reply:
This and `-f ` options are not supported anymore.
Probably there're more outdated options, I'll take a closer look later

Reply:
Ok